src/components/Book.js

```diff
@@ -3,40 +3,66 @@ import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import Button from './Button';
 import { removeBook, removeAsyncBook } from '../redux/books/booksSlice';
+import './styles/book.css';
 
-function Book({ id, title, author }) {
+function Book({
+  id, title, author, category,
+}) {
   const dispatch = useDispatch();
   const handelRemoveBook = () => {
     dispatch(removeAsyncBook(id));
     dispatch(removeBook(id));
   };
   return (
-    <>
-      <div>
-        <ul>
+    <div className="Lesson-Panel">
+      <p className="school-of">
+        {category}
+        {' '}
+      </p>
+      <h4 className="title">{title}</h4>
+      <p className="suzanne-collins">{author}</p>
+      <div className="flex-spac-btwn">
+        <div className="buttons-contanier flex">
+          <Button className="comments-btn" value="Comment" type="button" />
+          <hr />
+          <Button className="remove-btn" type="button" onClick={handelRemoveBook} value="Remove" />
+          <hr />
+          <Button className="edit-btn" value="Edit" type="button" />
+        </div>
+        <div className="progress-chapter  flex-spac-btwn">
+          <div className="percent-container flex ">
+            <div className="oval-2" />
+            <div className="percent-info">
+              <p className="percent-complete">100%</p>
+              <p className="completed">Completed</p>
+            </div>
+          </div>
 
-          <li id={id}>
-            <h4>
-              Title :
-              {title}
-            </h4>
-            <p>
-              Author :
-              {author}
-            </p>
-            <Button className="remove-book" type="button" value="Remove" onClick={handelRemoveBook} />
-          </li>
-
-        </ul>
+          <hr />
```

Comment:
Good work implementing the different styling, there are some changes before finishing the project 🏴󠁧󠁢󠁷󠁬󠁳󠁿 

- Kindly fix the grey line to apply the same style as on the Zeplin design
- Kindly change the font style to match Zeplin's design

Your design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/2a125827-80cf-4cf4-82b2-81313fb0f7ed)

Zeplin design

![image](https://github.com/naji-97/sec-book-store/assets/93630700/678d6519-2c21-4bb6-92a5-8370a01613c4)